docs/src/howto/howto_3.md

```diff
@@ -73,13 +73,15 @@ In addition to the distributions available in the `Distributions` package, Mimi
 
 ### Apply RVs to model parameters
 
+**For all applications in this section, it is important to note that for each trial, a random variable on the right hand side of an assignment  will take on the value of a *single* draw from the given distribution.  This means that even if the random variable is applied to more than one parameter on the right hand side (such as assigning to a slice), each of these parameters will be assigned the same value, not different draws from the distribution.**
+
 The macro next defines how to apply the values generated by each RV to model parameters based on a pseudo-assignment operator:
 
 - `param = RV` replaces the values in the parameter with the value of the RV for the current trial.
-- `param += RV` replaces the values in the parameter with the sum of the original value and the value of the RV for the current trial.
-- `param *= RV` replaces the values in the parameter with the product of the original value and the value of the RV for the current trial.
+- `param += RV` replaces the values in the parameter with the sum of the previous trial's value and the value of the RV for the current trial.
+- `param *= RV` replaces the values in the parameter with the product of the previous trial's value and the value of the RV for the current trial.
```

Comment:
I don't think this is correct? I think it is the product of the original value and the random variable? not sure though. But that would be mean it shouldn't be thought of as a random walk afterall